python/dataproc_templates/snowflake/README.md

```diff
@@ -0,0 +1,133 @@
+## 1. Snowflake To GCS
+
+General Execution:
+
+```
+export GCP_PROJECT=<gcp-project-id>
+export REGION=<gcp-project-region>
+export GCS_STAGING_LOCATION=<gcs-bucket-staging-folder-path>
+export SUBNET=<gcp-project-dataproc-clusters-subnet>
+bin/start.sh \
+-- \
+--template=SNOWFLAKETOGCS \
+--snowflake.to.gcs.sf.url=<snowflake-account-url> \
+--snowflake.to.gcs.sf.user=<snowflake-user> \
+--snowflake.to.gcs.sf.password=<snowflake-user-password> \
+--snowflake.to.gcs.sf.database=<snowflake-database> \
+--snowflake.to.gcs.sf.schema=<snowflake-schema> \
+--snowflake.to.gcs.sf.warehouse=<snowflake-warehouse> \
+--snowflake.to.gcs.sf.query=<snowflake-select-query> \
+--snowflake.to.gcs.output.location=<gcs-output-location> \
+--snowflake.to.gcs.output.format=<csv|avro|orc|json|parquet> \
+--snowflake.to.gcs.output.mode=<Overwrite|ErrorIfExists|Append|Ignore> \
+--snowflake.to.gcs.partition.column=<gcs-output-partitionby-columnname> \
+--snowflake.gcs.sf.autopushdown=<on|off>
+```
+
```

Comment:
Add Required JAR files section, and explain how to get the snowflake-jdbc-3.13.14.jar and the spark-snowflake_2.12-2.10.0-spark_3.1.jar